package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zimbra/zimlet-cli",
-  "version": "5.9.0",
+  "version": "5.10.0",
```

Comment:
Version bumping can be done by the publisher, so this can be left alone on this PR.